cola/widgets/toolbar.py

```diff
@@ -0,0 +1,670 @@
+# encoding: utf-8
+from __future__ import division, absolute_import, unicode_literals
+
+import copy
+
+from cola import cmds
+from cola import guicmds
+from cola.qtutils import app
+from cola.widgets import action
+from cola.widgets import browse
+from cola.widgets import compare
+from cola.widgets import createbranch
+from cola.widgets import createtag
+from cola.widgets import editremotes
+from cola.widgets import finder
+from cola.widgets import grep
+from cola.widgets import merge
+from cola.widgets import patch
+from cola.widgets import recent
+from cola.widgets import remote
+from cola.widgets import search
+from cola.widgets import standard
+from cola.widgets import stash
+from qtpy import QtGui
+from qtpy.QtCore import Qt
+from qtpy import QtWidgets
```

Comment:
qtpy imports should come before cola here (separated by a newline) and we should probably prefer relative imports these days.  We've eliminated all of the non-essential absolute cola imports so new code should follow that style